src/mixins/portfolio.js

```diff
@@ -136,6 +147,28 @@ export const createPorfolioMixin = ({
         shouldApplyDisplayState,
       });
     },
+    nftCreatorAddressListOfCollected() {
+      return [
+        ...new Set(
+          this.nftClassListOfCollected
```

Comment:
This should exclude PNFT?  
I have strange behavior in [my portfolio](https://rinkeby.liker.land/like1f2k6t98t9kjpdh9v9y95zhmjewuevmuwh82vj0?tab=collected) when filtering 角醒
```suggestion
          this.nftClassListOfCollectedExcludedOther
```